data/json/items/magazine/762.json

```diff
@@ -125,7 +125,7 @@
     "looks_like": "stanag30",
     "type": "MAGAZINE",
     "name": { "str": "vz. 58 30-round magazine" },
-    "description": "An original surplus Czechoslovak magazine, constructed out of lightweight aluminum and made for use within vz. 58 pattern rifles and modern derivatives.  Despite strongly resembling standard Kalashnikov magazines, this curved box magazine is unsuitable for insertion into AK patten mag wells due to changes made in its follower assembly to facilitate a last-round-hold-open feature within the vz. 58.",
+    "description": "An original surplus Czechoslovak magazine, constructed out of lightweight aluminum and made for use within vz. 58 pattern rifles and modern derivatives.  Despite strongly resembling standard Kalashnikov magazines, this curved box magazine is unsuitable for insertion into AK-pattern mag wells due to changes made in its follower assembly to facilitate a last-round-hold-open feature within the vz. 58.",
```

Comment:
Czechoslovakian here as well

Reply:
No, for reasons described in the other comment